conan/tools/files/files.py

```diff
@@ -341,18 +342,24 @@ def print_progress(_, __):
                 except Exception as e:
                     output.error("Error extract %s\n%s" % (file_.filename, str(e)))
         else:  # duplicated for, to avoid a platform check for each zipped file
+            file_timestamps = []
             for file_ in zip_info:
                 extracted_size += file_.file_size
                 print_progress(extracted_size, uncompress_size)
                 try:
                     z.extract(file_, full_path)
+                    file_path = os.path.join(full_path, file_.filename)
+                    ts = time.mktime(file_.date_time + (0, 0, -1))
```

Comment:
Why this 0, 0, -1? This would need some comment

Reply:
`file_.date_time` provides [six timestamp fields](https://docs.python.org/3/library/zipfile.html#zipfile.ZipInfo.date_time), but `time.mktime()` needs three additional fields to make a [`struct_time`](https://docs.python.org/3/library/time.html#time.struct_time).

I can add a comment to this effect.